gravitee-apim-console-webui/src/management/settings/portal-ng/portal-ng.component.spec.ts

```diff
@@ -0,0 +1,1148 @@
+          forceLogin: {
+            enabled: false,
+          },
+          github: {},
+          google: {
+            clientId: 'testClientId',
+          },
+          localLogin: {
+            enabled: true,
+          },
+          oauth2: {},
+        },
+        company: {
+          name: 'Gravitee22',
+        },
+        cors: {
+          allowHeaders: ['Cache-Control', 'Pragma'],
+          allowMethods: ['GET', 'DELETE'],
+          allowOrigin: ['test.entrypoint.dev', 'test.entrypoint.dev2'],
+          exposedHeaders: ['ETag', 'X-Xsrf-Token'],
+          maxAge: 1728000,
+        },
+        dashboards: {
+          apiStatus: {
+            enabled: true,
+          },
+        },
+        documentation: {
+          url: 'https://docs.gravitee.ios',
+        },
+        email: {
+          enabled: true,
+          from: 'test@email',
+          host: 'testhost',
+          password: '********',
+          port: 1,
+          properties: {
+            auth: true,
+            sslTrust: 'testssl',
+            startTlsEnable: false,
+          },
+          protocol: 'smtp',
+          subject: '[gravitee] %s',
+          username: 'testusername',
+        },
+        openAPIDocViewer: {
+          openAPIDocType: {
+            defaultType: 'Redoc',
+            swagger: {
+              enabled: false,
+            },
+          },
+        },
+        plan: {
+          security: {
+            apikey: {
+              enabled: true,
+            },
+            customApiKey: {
+              enabled: true,
+            },
+            jwt: {
+              enabled: true,
+            },
+            keyless: {
+              enabled: false,
+            },
+            oauth2: {
+              enabled: true,
+            },
+            push: {
+              enabled: true,
+            },
+            sharedApiKey: {
+              enabled: false,
+            },
+          },
+        },
+        portal: {
+          analytics: {
+            enabled: true,
+            trackingId: 'tests',
+          },
+          apikeyHeader: 'Test company',
+          apis: {
+            tilesMode: {
+              enabled: true,
+            },
+          },
+          entrypoint: 'test.entrypoint',
+          rating: {
+            comment: {
+              mandatory: false,
+            },
+            enabled: true,
+          },
+          support: {
+            enabled: true,
+          },
+          tcpPort: 4082,
+          uploadMedia: {
+            enabled: true,
+            maxSizeInOctet: 200000,
+          },
+          url: 'test',
+          userCreation: {
+            enabled: false,
+          },
+        },
+        reCaptcha: {
+          enabled: false,
+        },
+        scheduler: {
+          notifications: 101,
+          tasks: 10,
+        },
+      });
+    });
+
+    it('display settings form and edit Console fields', async () => {
+      expectPortalSettingsGetRequest(portalSettingsMock);
```

Comment:
You can move repeated code to the beforeEach block (i.e. get request, save bar is visible to be false)

Reply:
Done